packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -5153,9 +5178,39 @@ export class WalletService {
     return sign;
   }
 
+
+  changellySignRequestsV2(message, secret: string) {
+    if (!message || !secret) throw new Error('Missing parameters to sign Changelly v2 request');
+
+    const privateKey = crypto.createPrivateKey({
+      key: secret,
+      format: 'der',
+      type: 'pkcs8',
+      encoding: 'hex'
+    });
+    
+    const publicKey = crypto.createPublicKey(privateKey).export({
+        type: 'pkcs1',
+        format: 'der'
+    });
+
+    const signature = crypto.sign('sha256', Buffer.from(JSON.stringify(message)), {
+      key: privateKey,
+      type: 'pkcs8',
+      format: 'der'
```

Comment:
This key object is unnecessary. You can just pass privateKey by itself:
```
crypto.sign('sha256', ...., privateKey);
```

Reply:
Done! Ty!